devito/pointdata.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, *args, **kwargs):
             self.coordinates = DenseData(name='%s_coords' % self.name,
                                          dimensions=[self.indices[1], d],
```

Comment:
Ooops, sorry I misread. @navjotk is right, it's the number of points (arbitrary default `p`), while `d` is the spatial dimension.